proposals/dao/tip_123.ts

```diff
@@ -0,0 +1,109 @@
+  console.log('Fei Labs TRIBE burner deployed to: ', feiLabsTribeBurner.address);
+
+  return {
+    daoTimelockBurner,
+    deprecatedRariFeiTimelockBurner,
+    deprecatedRariTribeTimelockBurner,
+    feiLabsTribeBurner
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  pcvStatsBefore = await contracts.collateralizationOracle.pcvStats();
+};
+
+// Tears down any changes made in setup() that need to be
+// cleaned up before doing any validation checks.
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in teardown for fip${fipNumber}`);
+};
+
+// Run any validations required on the fip using mocha or console logging
+// IE check balances, check state of contracts, etc.
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  // display pcvStats
+  console.log('----------------------------------------------------');
+  console.log(' pcvStatsBefore.protocolControlledValue [M]e18 ', Number(pcvStatsBefore.protocolControlledValue) / 1e24);
+  console.log(' pcvStatsBefore.userCirculatingFei      [M]e18 ', Number(pcvStatsBefore.userCirculatingFei) / 1e24);
+  console.log(' pcvStatsBefore.protocolEquity          [M]e18 ', Number(pcvStatsBefore.protocolEquity) / 1e24);
+  const pcvStatsAfter: PcvStats = await contracts.collateralizationOracle.pcvStats();
+  console.log('----------------------------------------------------');
+  console.log(' pcvStatsAfter.protocolControlledValue  [M]e18 ', Number(pcvStatsAfter.protocolControlledValue) / 1e24);
+  console.log(' pcvStatsAfter.userCirculatingFei       [M]e18 ', Number(pcvStatsAfter.userCirculatingFei) / 1e24);
+  console.log(' pcvStatsAfter.protocolEquity           [M]e18 ', Number(pcvStatsAfter.protocolEquity) / 1e24);
+  console.log('----------------------------------------------------');
+  const pcvDiff = pcvStatsAfter.protocolControlledValue.sub(pcvStatsBefore.protocolControlledValue);
+  const cFeiDiff = pcvStatsAfter.userCirculatingFei.sub(pcvStatsBefore.userCirculatingFei);
+  const eqDiff = pcvStatsAfter.protocolEquity.sub(pcvStatsBefore.protocolEquity);
+  console.log(' PCV diff                               [M]e18 ', Number(pcvDiff) / 1e24);
+  console.log(' Circ FEI diff                          [M]e18 ', Number(cFeiDiff) / 1e24);
+  console.log(' Equity diff                            [M]e18 ', Number(eqDiff) / 1e24);
+  console.log('----------------------------------------------------');
+
+  // 0. Assert overcollaterised
+  // expect(await contracts.collateralizationOracle.isOvercollateralized()).to.be.true;
+
+  // 1. Verify Fei DAO timelock admin burned
+  expect(await contracts.feiDAOTimelock.admin()).to.equal(addresses.daoTimelockBurner);
```

Comment:
to be extra safe, would be good to verify that even succeeded votes on the DAO can't be executed. They probably can't even be queued.

Also good to verify that the number of addresses with governor role is 0, there is a method on the OZ AccessControlEnumerated which can check this.

Reply:
Verified num GOVERNORs is 0, and also verified num of addresses with other major roles is as expected

Reply:
Added test to verify can not queue